snuba/web/query.py

```diff
@@ -197,8 +188,6 @@ def raw_query(
         request, sql, timer, stats, "success", query_settings
     )
 
-    result["timing"] = timer
```

Comment:
Do you know why this was even happening ?

Reply:
I'm not really sure, to be honest. I preserved the behavior in `run_query` but am not against removing it in the future.